src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/mappers/AddressesMapper.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.transformation.mappers;
+
+import uk.gov.hmcts.reform.bsp.common.model.shared.in.OcrDataField;
+
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.bsp.common.mapper.AddressMapper.applyMappings;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.helper.CommonConditions.isApplicantRepresented;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.helper.CommonConditions.isRespondentRepresented;
+
+/**
+ * Mapping fields related to addresses (applicant, respondent and applicant solicitor, respondent solicitor).
+ *
+ * <p>We need it because we want to map to fields CCDConfigConstant.APPLICANT_ADDRESS
+ * and CCDConfigConstant.RESPONDENT_ADDRESS different set of values. it may be either address details
+ * of citizen (if not represented) or their solicitor (if represented).
+ */
+public class AddressesMapper {
+
+    public static class CcdFields {
+        public static final String APPLICANT = "applicantAddress";
+        public static final String RESPONDENT = "respondentAddress";
+    }
+
+    public static class TempCcdFields {
+        public static final String APPLICANT_SOLICITOR = "applicantSolicitorAddress";
+        public static final String RESPONDENT_SOLICITOR = "rSolicitorAddress";
```

Comment:
Why not use the constants in CCDConfigConstant file? "RESP_SOLICITOR_ADDRESS" ?

Reply:
I wanted to keep it close to area. I may just to do:

```
        public static final String RESPONDENT_SOLICITOR = CCDConfigConstant .RESP_SOLICITOR_ADDRESS;
```

Reply:
@johnoneill46 , here:
https://github.com/hmcts/bsp-common-lib/pull/21